packages/benchmark/src/benchmarkerator.js

```diff
@@ -45,6 +49,7 @@ import { makeWalletFactoryDriver } from '@agoric/boot/tools/drivers.ts';
  *    options: Record<string, string>,
  *    argv: string[],
  *    actors: Record<string, import('@agoric/boot/tools/drivers.ts').SmartWalletDriver>,
+ *    tools: Record<string, unknown>,
```

Comment:
these are known. Please type them. You could have a `makeTools()` function and make this have type `ReturnType<typeof makeTools>`